gpt4all-backend/gptj.cpp

```diff
@@ -871,28 +880,23 @@ int32_t GPTJ::threadCount() {
     return d_ptr->n_threads;
 }
 
-GPTJ::~GPTJ()
-{
+GPTJ::~GPTJ() {
```

Comment:
Would also prefer this in sep. PR. basically, I would like this PR to be laser focused on only the stuff that actually matters for completing. Will help to review

Reply:
Understood.